fsh-generated/resources/Bundle-zbj-bundle-questionnaires.json

```diff
@@ -591,29 +591,53 @@
             ]
           },
           {
-            "linkId": "da65ffd8-fb90-4918-becf-dfe0439577d8",
+            "linkId": "245f3b7e-47d2-4b78-b751-fb04f38b17b9",
             "text": "Protocol",
             "type": "choice",
             "required": true,
             "answerOption": [
               {
-                "valueString": "Instel / titratiefase",
+                "valueCoding": {
+                  "code": "195967001",
+                  "system": "http://snomed.info/sct",
+                  "display": "Instel / titratiefase"
+                },
                 "initialSelected": true
               },
               {
-                "valueString": "Astma - wekelijks"
+                "valueCoding": {
+                  "code": "195967001",
```

Comment:
I think, canonically speaking, all answers are now the same since they have the same coding. I don't think that's right @bramwesselo ?